sympy/sets/sets.py

```diff
@@ -990,6 +990,16 @@ def _union(self, other):
 
         See Set._union for docstring
         """
+        if isinstance(other, Complement):
+            A, B = other.args
+            if A == S.UniversalSet:
```

Comment:
Whenever you write "== S." you can replace that with "is S." so `A is S.UniversalSet`. The same below, too.